docs/contributing/code-review.md

```diff
@@ -1,235 +1,118 @@
-   Apply slightly higher code review scrutiny when reviewing changes to the authentication workflow.
-   :::
-1. **Merge the change**. Once approved, merge the PR. This will deploy your change to all the hubs.
+1. Is this PR in service of moving forward on a issue that you picked up
+   from the Engineering Board?
 
-(infrastructure:review:terraform)=
-## Terraform changes
+If the answer to this is "No", consider if you really *must* do this. The
+engineering board is a representation of the work the *team* has committed to
+doing. Ask yourself if you can instead be doing something else that can move
+this forward. Can this wait, and go through the refinement / sprint planning
+process?
 
-[Terraform](tools:terraform) is used for directly provisioning and modifying cloud infrastructure (as opposed to just deploying applications on pre-existing cloud infrastructure).
-This lets us treat our infrastructure as code, so we can use regular code quality practices (code review, linters, extensive documentation, etc) to keep the quality of our infrastructure high.
-However, it comes with a few caveats:
+See [the guiding principles](https://compass.2i2c.org/engineering/workflow/1-week-iteration-workflow/#guiding-principle-move-the-team-forward)
+document for more guidance on what else you can do.
 
-1. Locally and iteratively testing terraform is impossible - as we
-   develop it, the infrastructure is deployed / modified / destroyed
-   as we go.
-2. Behavior of terraform code differs based on the current
-   state of the infrastructure (as represented in the
-   [state file](https://www.terraform.io/docs/language/state/index.html)).
-3. While `terraform plan` can tell us what terraform *thinks* it is
-   going to do, it might not always succeed. Permission errors or
-   organizational restrictions can cause it to fail. Timeouts can happen.
-   Runtime errors pop up.
+You should break this rule when it makes sense, but do not do so
+**casually**. This is an important mechanism by which we can build a
+team we can all trust in over
 
-Thanks to all these, it is impossible to either:
+This may make you uncomfortable, and that is ok. Ultimately our goal is to
+build a **team we can all trust in**, and this is a very important part
+of getting there.
 
-- Accurately gauge the impact of a terraform code change just from looking at the text
-- Automatically deploy it via continuous deployment
+## After making the PR
 
-As a result, we have some workflows that are specific to using Terraform, described below.
+After you make your PR, ask yourself these three questions:
 
-### Changes to running infrastructure
+1. *If* this PR breaks something, do you feel comfortable debugging it **or**
+   reverting it to undo the change?
+2. Do you know how to *verify* the effects of merging this PR?
 
-In this scenario, there is already-running infrastructure and you are deploying changes to it.
+If the answer to all these 2 questions are 'yes', you should go ahead
```

Comment:
Again, I would exclude the "2" here so we can add to the list of questions without needing to update other language.